docs/tutorial.rst

```diff
@@ -317,6 +317,8 @@ section. We can then consume it through templates and assign it to bash
 variables. Because we now have access to the bucket in the ConfigMap, we can
 also push the log to S3. In this Secret, we store the credentials as binary
 data. We can use the templating engine ``toString`` and ``quote`` functions, courtesy of sprig.
+If you are not using AWS S3 as the Object Storage, you need to add ``--endpoint {{ .Profile.Location.Endpoint }}``
+to the end of the aws command so it can connect to the propper S3 object storage.
```

Comment:
If we are changing command just for the minio storage, let's have this after the blueprint manifest as a `note`, and there we will mention the updated `aws` command.

Reply:
```suggestion
If you are not using AWS S3 as the Object Storage, you need to add ``--endpoint {{ .Profile.Location.Endpoint }}``
to the end of the aws command so it can connect to the appropriate S3 object storage.
```
Agree with @viveksinghggits about having a separate Note. 
This will hold true for all non-aws S3 compatible storage.